src/mod_roster.erl

```diff
@@ -573,26 +582,30 @@ ask_to_pending(subscribe) -> out;
 ask_to_pending(unsubscribe) -> none;
 ask_to_pending(Ask) -> Ask.
 
--spec in_subscription(Acc :: mongoose_acc:t(),
-                      ToJID :: jid:jid(),
-                      FromJID :: jid:jid(),
-                      Type :: sub_presence(),
-                      Reason :: iodata()) ->
-    mongoose_acc:t().
-in_subscription(Acc, ToJID, FromJID, Type, Reason) ->
-    HostType = mongoose_acc:host_type(Acc),
+-spec in_subscription(Acc, Params, Extra) -> {ok, Acc} when
+    Acc :: mongoose_acc:t(),
+    Params :: #{to_jid := jid:jid(),
+                from_jid := jid:jid(),
+                type := mod_roster:sub_presence(),
```

Comment:
This is the same module where it is used, the fully-qualified name is not needed. Same in the function below